wandb/sdk/internal/meta.py

```diff
@@ -38,9 +38,9 @@ def __init__(self, settings=None, interface=None):
         self.fname = os.path.join(self._settings.files_dir, METADATA_FNAME)
         self._interface = interface
         self._git = GitRepo(
-            remote=self._settings["git_remote"]
-            if "git_remote" in self._settings.keys()
-            else "origin"
+            remote=self._settings.get("git_remote", None),
```

Comment:
why arent the settings just referenced like this:
```
self.settings.git_remote
```

Reply:
Good catch will be fixed!